c/ldv-regression/test27-2.c

```diff
@@ -27,7 +27,7 @@ int main()
   i = __VERIFIER_nondet_int();
   for (int j = 0; j < 10; j++) {
       a[j] = __VERIFIER_nondet_int();
-      dummies[j] = __VERIFIER_nondet_pointer();
+      dummies[j] = 0;
```

Comment:
Could you please have a look at it again? I do not understand why replacing a call to ```__VERIFIER_nondet_pointer``` by ```0``` is OK.
 
Same applies for ```test27-1.c```.

Reply:
test27-2.c isn't memory safe (neither was memory safe before nor is now), and test27-1.c is memory safe irrespective of the value used here as that value is never read. I've added comments to both files.